client/superdesk.config.js

```diff
@@ -8,21 +8,15 @@ module.exports = function(grunt) {
     return {
         apps: [
             'superdesk.analytics',
-            'superdesk-planning'
+            'superdesk-planning',
+            'belga',
         ],
         importApps: [
             'superdesk-analytics',
-            'superdesk-planning'
+            '../belga',
```

Comment:
There is a js error:

![screenshot 2019-01-14 at 14 51 26](https://user-images.githubusercontent.com/4199922/51116608-e6f47c80-180b-11e9-8b73-9efbde108907.png)


I think it's because of `superdesk-planning` was removed from `importApps`.